docs/design/gatewayapi-translator.md

```diff
@@ -1,13 +1,13 @@
 # Gateway API Translator Design
 
 ## Assumptions
-- initially target core conformance features only, to be followed by extended conformance features
+- Initially target core conformance features only, to be followed by extended conformance features
 
 ## Inputs and Outputs
 
 The main inputs to the Gateway API translator are:
 - the GatewayClass to process
-- Gateways, HTTPRoutes, Services, Secrets
+- Gateways, HTTPRoutes, Services, Secrets, ReferenceGrants
```

Comment:
@chauhanshubham TLSRoutes should be added as part of your TLS pass through work.